pkg/models/domain.go

```diff
@@ -0,0 +1,24 @@
+// (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+package models
+
+type GetAllDomains struct {
+	NetworkDomains []GetDomain `json:"networkDomains"`
+
+	NetworkDomainCount int  `json:"networkDomainCount"`
+	Meta               Meta `json:"meta"`
```

Comment:
Do we not need the sub elements for Meta? It is a struct obj.

Reply:
Meta object is not part of terraform schema. So no need to include tf flag with sub here.